core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -343,6 +343,15 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     value
   }
 
+  /** Control our logLevel. This overrides any user-defined log settings.
+   * @param logLevel The desired log level as a string.
+   * Valid log levels include: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN
+   * Invalid log level defaults to DEBUG
```

Comment:
Do you want this instead of the default to debug behavior that is currently present?